cypress/e2e/with_mock_data/items.cy.ts

```diff
@@ -636,6 +636,70 @@ describe('Items', () => {
       cy.findByLabelText('Show error details').should('exist');
       cy.findByText('Upload failed').should('exist');
     });
+
+    it('should render in table headers', () => {
+      cy.findByText('5YUQDDjKpz2z').click();
+      cy.findByText(
+        'High-resolution cameras for beam characterization. 1'
+      ).should('exist');
+
+      cy.findByText('Attachments').click();
+      cy.findByText('Filename').should('be.visible');
+      cy.findByText('Title').should('be.visible');
+      cy.findByText('Description').should('be.visible');
+      // Scroll 750 pixels to the right, so created and last modifed headers are visible.
+      // scrollIntoView() wasn't working in headless mode.
+      cy.scrollTo(750, 0)
+      cy.findByText('Created').should('be.visible');
+      cy.findByText('Last modified').should('be.visible');
```

Comment:
```suggestion
      cy.findByText('Last modified').scrollIntoView();
      cy.findByText('Created').should('be.visible');
      cy.findByText('Last modified').should('be.visible');
```

Change this once #1270 is merged into develop